buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -279,7 +253,8 @@ RUN cd /tmp \
   && cd .. \
   && rm -rf openssl-OpenSSL_1_1_1b \
   && echo "/usr/local/openssl-1.1.1b/lib" > /etc/ld.so.conf.d/openssl-1.1.1b.conf \
-  && echo "PATH=/usr/local/openssl-1.1.1b/bin:$PATH" > /etc/environment
+  && echo "PATH=/usr/local/openssl-1.1.1b/bin:$PATH" >> /etc/environment \
```

Comment:
Writing to `/etc/environment` doesn't seem to do anything - please remove this.

Reply:
See my comment above about `/etc/environment`.

Reply:
Same as I commented above `/etc/environment` is similar to `~/.ssh/environment` except it applies to all users. Which implementation would you prefer?

Reply:
I think we should use something that is documented. Can you share a link to documentation that supports that expectation? As I said, I couldn't find anything.

Reply:
https://help.ubuntu.com/community/EnvironmentVariables#System-wide_environment_variables

Reply:
That information is for Ubuntu (and perhaps more specifically, for bash): it doesn't appear to apply to centos (I tried).